scripts/install

```diff
@@ -141,45 +29,31 @@ chown -R $app:www-data "$install_dir"
 #=================================================
 ynh_script_progression --message="Configuring NGINX web server..." --weight=1
 
-### `ynh_add_nginx_config` will use the file conf/nginx.conf
-
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
+#=================================================
+# ADD A CONFIGURATION
+#=================================================
+ynh_script_progression --message="Adding a configuration file..." --weight=1
+
+ynh_add_config --template="../conf/.env" --destination="$install_dir/.env"
+
+chmod 400 "$install_dir/.env"
+chown $app:$app "$install_dir/.env"
+
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 # Configuration files
 #=================================================
 
-config="$install_dir/.config"
-env_file="$install_dir/.env"
-
-export MIX_ENV=prod FLAVOUR=classic
-
 ynh_exec_warn_less just config
-# generate secrets
-ynh_replace_string --match_string="SECRET_KEY_BASE=you-should-put-a-secure-string-here" --replace_string="SECRET_KEY_BASE=$(openssl rand -base64 128)" --target_file="$env_file"
-ynh_replace_string --match_string="SIGNING_SALT=you-should-put-a-different-secure-string-here" --replace_string="SIGNING_SALT=$(openssl rand -base64 128)" --target_file="$env_file"
-ynh_replace_string --match_string="ENCRYPTION_SALT=you-should-put-yet-another-secure-string-here" --replace_string="ENCRYPTION_SALT=$(openssl rand -base64 128)" --target_file="$env_file"
-
-# Configure server ports
-ynh_replace_string --match_string="HOSTNAME=localhost" --replace_string="HOSTNAME=$domain" --target_file="$env_file"
-# TODO : mail service ?
-ynh_replace_string --match_string="SERVER_PORT=4000" --replace_string="SERVER_PORT^=$port" --target_file="$env_file"
-ynh_replace_string --match_string="PUBLIC_PORT=4000" --replace_string="PUBLIC_PORT=443" --target_file="$env_file"
-
-# TODO : Configure S3 - with proper Yunohost question during installation
-# UPLOADS_S3_BUCKET=
-# UPLOADS_S3_ACCESS_KEY_ID=
-# UPLOADS_S3_SECRET_ACCESS_KEY=
-
-# max file upload size
-UPLOAD_LIMIT="${media_upload_size:0:2}000000" # convert the MB argument in bytes
```

Comment:
Why removing this ?

Reply:
IMO it is better to set a `.env` file

Reply:
I don't know how to convert this to something that could be included in such a file.

Reply:
https://github.com/YunoHost-Apps/bonfire_ynh/blob/cleaning/conf/.env

Reply:
Yes but I don't know how to set that parameter.

Reply:
I think I found the way to do this.